jerry-core/jcontext/jcontext.c

```diff
@@ -38,7 +38,13 @@ jerry_context_t jerry_global_context;
 /**
  * Global heap.
  */
-jmem_heap_t jerry_global_heap __attribute__ ((aligned (JMEM_ALIGNMENT))) JERRY_GLOBAL_HEAP_SECTION;
+#ifdef _MSC_VER
+#define _memory_alignment
+#else
+#define _memory_alignment __attribute__((aligned(JMEM_ALIGNMENT)))
+#endif
```

Comment:
This should definitely be moved to and handled by jrt.h. And renamed to match existing naming scheme. Something like:

```c
#define __attr_aligned___(ALIGNMENT) __attribute__((aligned(ALIGNMENT)))
```

Reply:
moved